### PR TITLE
Enable OTP verification per login

### PR DESCRIPTION
**Pull request by jurakp**

### CODE REVIEW

**Review by krisrjohnson21 — APPROVED** [review]
🧼

File: files/modules/mbta-keycloak-browser-authenticator-1.2.1.jar
Comment (paulswartz) [inline files/modules/mbta-keycloak-browser-authenticator-1.2.1.jar:1]: question: is the code for this JAR checked in somewhere we can track?

File: files/keycloak-modules/org.keycloak.keycloak-model-infinispan-22.0.5.jar
Comment (paulswartz) [inline files/keycloak-modules/org.keycloak.keycloak-model-infinispan-22.0.5.jar:1]: question: did we mean to remove this?

File: files/modules/mbta-keycloak-browser-authenticator-1.2.1.jar
Comment (jurakp) [reply files/modules/mbta-keycloak-browser-authenticator-1.2.1.jar:1]: Yes, it is https://github.com/mbta/keycloak-browser-authenticator tag R-1.2.1 as usual.

File: files/keycloak-modules/org.keycloak.keycloak-model-infinispan-22.0.5.jar
Comment (jurakp) [reply files/keycloak-modules/org.keycloak.keycloak-model-infinispan-22.0.5.jar:1]: Yes, it is not needed now.

**Review by paulswartz — APPROVED** [review]

**Pull request merged**